frappe/email/smtp.py

```diff
@@ -247,6 +247,8 @@ def sess(self):
 
 		except _socket.error as e:
 			# Invalid mail server -- due to refusing connection
+                        #unknown mail server due to connection error
+                        #this gives a better error code
```

Comment:
Thanks for your contribution @kelvinace1 

However, I feel the previous comment already covers this. If you think there's more to this, I'd suggest getting more verbose describing it here.

Also, please do checkout the contribution guidelines defined [here](https://github.com/frappe/erpnext/wiki/Contribution-Guidelines)